app/middleware/createGpViewModel.js

```diff
@@ -34,6 +35,7 @@ function createGpViewModel(req, res, next) {
       facilities: facilitiesMapper(gpData.facilities),
       services: servicesMapper(gpData.services),
       openingTimes: openingTimesMapper(gpData.openingTimes),
+      exceptionalTimes: exceptionalTimesMapper(gpData.openingTimes),
```

Comment:
The openingTimesMapper and the exceptionalTimesMapper both take the same parameter, which makes me think the logic should be performed in same function.

The source data is structured as 
```
openingTimes: {
  reception: ...,
  surgery: ..., 
  alterations: ...
}
```
exceptiontalTimes could be a subfield on openingTimes if it was all done in the same function